CLI.md

```diff
@@ -1022,6 +1022,7 @@ $ zanshin organization scan_target onboard_aws_organization [OPTIONS] REGION ORG
 * `--target-accounts [ALL|MASTER|MEMBERS|NONE]`: choose which accounts to onboard
 * `--exclude-account TEXT`: ID, Name, E-mail or ARN of AWS Account not to be onboarded  [default: ]
 * `--boto3-profile TEXT`: Boto3 profile name to use for Onboard AWS Account  [default: default]
+* `--no-boto3-profile TEXT`: Won't use a boto3 profile. Useful if using Zanshin CLI on EC2 instances or Lambdas  [default: False]
```

Comment:
Why not simply make `--boto3-profile` optional and handle that in the code? I'm not sure having a separate `--no-boto3-profile` is the best option.

Reply:
So this will change the current behavior. Currently if `--boto3-profile` is not informed, we use the `default` profile.

If we implement the way you suggest (that I agree is much better), we'll change this behavior. @asieira Is this acceptable?